src/app/_constants/sortConstants.ts

```diff
@@ -1,10 +1,46 @@
-import { type SortSetting } from '@/types/sortTypes'
+import { SortSetting } from '@/types/sortTypes'
 
-export const sortSettings: SortSetting[] = [
-  { id: 'newest', name: 'Newest' },
-  { id: 'oldest', name: 'Oldest' },
+const sortOptions = {
+  cronologicalSort: {
+    default: 'newest' as const,
+    values: [
+      { id: 'newest', name: 'Newest' },
+      { id: 'oldest', name: 'Oldest' },
+    ] as SortSetting[],
+  },
+  alphabeticalSort: {
+    default: 'a-z' as const,
+    values: [
+      { id: 'a-z', name: 'Alphabetical (A-Z)' },
+      { id: 'z-a', name: 'Alphabetical (Z-A)' },
+    ] as SortSetting[],
+  },
+}
+
```

Comment:
I feel like this could be more strictly typed, the `as SortSetting[]` cast is a workaround.

Maybe using a global `as const` would be more accurate? I haven't tested it though, it's quite hard at the review stage :)

```ts
const sortOptions = {
  cronologicalSort: {
    default: 'newest',
    values: [
      { id: 'newest', name: 'Newest' },
      { id: 'oldest', name: 'Oldest' },
    ],
  },
  alphabeticalSort: {
    default: 'a-z',
    values: [
      { id: 'a-z', name: 'Alphabetical (A-Z)' },
      { id: 'z-a', name: 'Alphabetical (Z-A)' },
    ],
  },
} as const
```

Reply:
Ah yes, we can make it immutable, good one. 👍

Reply:
Is there a benefit to make it more strict? We use the values to get SortSettings, the `id` is types with type `SortOption`, not sure if the `name` needs to be strictly typed?